modules/@apostrophecms/rich-text-widget/ui/apos/components/ApostropheTiptapStyles.vue

```diff
@@ -1,20 +1,51 @@
 <template>
-  <select :value="active()" @change="style">
-    <option v-for="style, i in options.styles" :value="i">{{ style.label }}</option>
-  </select>
+  <div class="apos-tiptap-select">
+    <select
+      :value="active"
+      @change="setStyle"
+      class="apos-tiptap-control apos-tiptap-control--select"
+    >
+      <option
+        v-for="(style, i) in options.styles"
+        :value="i"
+        :key="style.label + '-' + i"
```

Comment:
Will there be more than one option for each style? Why add the index?

Reply:
habit, removed